pgmpy/factors/CPD.py

```diff
@@ -112,6 +112,27 @@ def __init__(self, variable, variable_card, values,
 
         super().__init__(variables, cardinality, values.flatten('C'))
 
+    def replace_variables(self, var1, var2):
+        new_scope = self.scope()
+        new_scope.reverse()
```

Comment:
@ankurankan  @abinashpanda  yes, the above one is cool but I have some inconsistent behavior when ```[::-1]``` is used to reverse the list. I have to swap some values in ```new_scope``` but this inturn changes ```self.scope()``` too if used with ```[::-1]```. Something very similar happened with ```new_cardinality``` too. Please refer to the comment I did below.